app/repositories/application/GeneralApplicationRepository.scala

```diff
@@ -890,7 +889,7 @@ class GeneralApplicationMongoRepository(timeZoneService: TimeZoneService,
   def extract(key: String)(root: Option[BSONDocument]) = root.flatMap(_.getAs[String](key))
 
   /*private def getAdjustmentsConfirmed(assistance: Option[BSONDocument]): Option[String] = {
```

Comment:
Is this commented fasttrack legacy? Do you think we can ditch the whole method now?